src/module.rs

```diff
@@ -121,7 +126,7 @@ where
         .map(|r| -> *const Value { &*r })
         .unwrap_or(null());
       unsafe { std::ptr::write(ret_ptr, r) }; // Write result to stack.
-      ret_ptr // Return stack pointer to the return value.
+      ret_ptr // Return caller-allocated memory pointer to the return value.
```

Comment:
You're right you're not the one introducing the verbiage. I'm okay with backing it out for now so it's at least consistent everywhere.